links-slackbot/src/main/java/juja/microservices/links/slackbot/Application.java

```diff
@@ -1,15 +1,41 @@
 package juja.microservices.links.slackbot;
 
-
+import org.apache.http.impl.client.HttpClients;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+
 
-/**
- * @author Danil Kuznetsov (kuznetsov.danil.v@gmail.com)
- */
 @SpringBootApplication
 public class Application {
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
Why do we need resttemplate ?

Reply:
I don't remember why somebody add all this methods in all Application main class. Maybe remove this and see what bad happens?

Reply:
We used manual bean for add additional configuration We don't need this resttemplate  We use feign client

Spring boot creates this bean automatically with default settings